src/g_canvas.c

```diff
@@ -649,24 +660,32 @@ int glist_isgraph(t_glist *x)
 
     /* This is sent from the GUI to inform a toplevel that its window has been
     moved or resized. */
-static void canvas_setbounds(t_canvas *x, t_float left, t_float top,
-                             t_float right, t_float bottom)
+static void canvas_setbounds(t_canvas *x, t_symbol * wm_state, t_float left, t_float top,
+                             t_float right, t_float bottom )
```

Comment:
i wouldn't change the interface for the canvas' `setbound` message like this.
probably just use `A_GIMME` and do the typechecking yourself
(this also removes the confusion of the order of arguments)

Reply:
Yes I realize that symbol argument needed to be passed at the beginning while my first choice was to pass it at the end.
Maybe if we move to a boolean (see my comment below) there would be a better/simpler type candidate for that (A_FLOAT ??)